src/main/java/com/epam/lowcost/controller/UserController.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.lowcost.controller;
+
+import com.epam.lowcost.model.User;
+import com.epam.lowcost.service.UserServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+@Controller()
+@RequestMapping(value = "/user")
+
+public class UserController {
+
+    @Autowired
+    UserServiceImpl userService;
```

Comment:
Why u use specific type ? Why not UserService as a interface ?

Reply:
Changed to interface.